packages/web-components/src/components/search/search.ts

```diff
@@ -119,6 +179,22 @@ class CDSSearch extends HostListenerMixin(FocusMixin(FormMixin(LitElement))) {
     }
   }
 
+  /**
+   * Move focus back to the magnifier element.
+   * Adds tabindex="-1" if it is not focusable yet.
+   */
+  private _focusMagnifier() {
+    const magnifier = this.shadowRoot?.querySelector(
+      `.${prefix}--search-magnifier`
+    ) as HTMLElement | null;
+    if (magnifier) {
+      if (!magnifier.hasAttribute('tabindex')) {
+        magnifier.setAttribute('tabindex', '-1');
```

Comment:
```suggestion
        magnifier.tabIndex = -1;
```

Can use the property directly here.